### PR TITLE
Added CFBundleShortVersionString for iOS framework

### PR DESCRIPTION
**Pull request by hiteshsondhi88**
This is required by app to be submitted to App Store

### CODE REVIEW

**Comment by hiteshsondhi88** [comment]
I signed it!

File: Source/Resources/GTLRiOSFramework-Info.plist [hunk 1]
```diff
@@ -12,6 +12,8 @@
 	<string>6.0</string>
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0.3</string>
```
Comment (thomasvl) [inline Source/Resources/GTLRiOSFramework-Info.plist]: Actually the version that [library api reports](https://github.com/google/google-api-objectivec-client-for-rest/blob/master/Source/Utilities/GTLRFramework.h#L27) isn't the same as the CocoaPod.  And hard coding this in the info.plist worries me a little as we'll likely forget to update it.

It probably makes sense to atleast get the api and framework version in sync.  I'm not sure if the pod version has to actually match.

Comment (thomasvl) [inline Source/Resources/GTLRiOSFramework-Info.plist]: Ok, then lets set this to 3.0.0 to match what the api point returns.  I've opened https://github.com/google/google-api-objectivec-client-for-rest/issues/53 to track automating all this going forward.

**Comment by customcoops77** [comment]
10:15-11:05

**Pull request merged**